webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-criterion-edit.js

```diff
@@ -163,6 +163,13 @@ export class SakaiRubricCriterionEdit extends RubricsElement {
     this.dispatchEvent(new CustomEvent('criterion-edited', {detail: {id: data.id, title: data.title, description: data.description}}));
     this.dispatchEvent(new SharingChangeEvent());
   }
+
+ openEditWithKeyboard(e){
```

Comment:
Could you format this function in line with the rest of the file? And, I think you meant to test two different codes here.